substrate/zombienet/0002-validators-warp-sync/test-validators-warp-sync.zndsl

```diff
@@ -31,11 +31,9 @@ bob: log line matches "Block history download is complete" within 120 seconds
 alice: reports block height is at least {{DB_BLOCK_HEIGHT}} within 10 seconds
 bob: reports block height is at least {{DB_BLOCK_HEIGHT}} within 10 seconds
 
-alice: reports substrate_beefy_best_block is at least {{DB_BLOCK_HEIGHT}} within 180 seconds
-bob: reports substrate_beefy_best_block is at least {{DB_BLOCK_HEIGHT}} within 180 seconds
-
-alice: reports substrate_beefy_best_block is greater than {{DB_BLOCK_HEIGHT}} within 180 seconds
-bob: reports substrate_beefy_best_block is greater than {{DB_BLOCK_HEIGHT}} within 180 seconds
+# In the worst case scenario, the validators should vote on 1 mandatory block each 6 seconds. And 1 era = 200 blocks.
```

Comment:
Not sure if I understand the math. Sorry for dumb question: one mandatory block is per era?